### DISCUSSION TITLE
GUI: simulation storage

### DISCUSSION THREAD

**Discussion by chenghuzi** [jonescompneurolab/hnn-core#508] [GSoC]
According to our proposal, we may need to consider the storage of simulation data now in GUI.  Which way, as you think, might be good to store simulation results? Should we also include a button for loading results just like loading parameter files?

**Comment by rythorpe** [comment]
> According to our proposal, we may need to consider the storage of simulation data now in GUI. Which way, as you think, might be good to store simulation results? 

As an [HDF5](https://docs.h5py.org/en/stable/) file that includes spiking, dipole, LFP/CSD data, etc. This should probably be built into the hnn-core API first, and then added to the GUI in a separate PR.

> Should we also include a button for loading results just like loading parameter files?

I think so!

**Comment by jasmainak** [comment]
I would keep this low on your list of priorities. We need a functional GUI that is documented and tested first. This is likely to open a can of worms with regards to storage format ... that's another GSoC altogether.

**Reply by rythorpe** [reply]
Yeah, I think you're right that this will blossom into a much larger project in and of itself.
